src/tests/integration/test-data/show/CSETest.test_identity(uint256,uint256).cse.expected

```diff
@@ -57,13 +57,11 @@
 module SUMMARY-TEST%CSETEST.TEST-IDENTITY(UINT256,UINT256):0
     
     
-    rule [BASIC-BLOCK-10-TO-12]: <foundry>
+    rule [BASIC-BLOCK-1-TO-15]: <foundry>
```

Comment:
Again, the diff is getting confused because there was a completely expected change in node enumeration, and so the order of the edges of the KCFG is different.